### PR TITLE
Minor/front end

### PR DESCRIPTION
**Pull request by venconi91**

### CODE REVIEW

File: api/sql/schema/340$transfer.orderByTT.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CREATE TYPE [transfer].[orderByTT] AS TABLE( --table type variable used for data sorting in stored procedures
```
Comment (kalinkrustev) [inline api/sql/schema/340$transfer.orderByTT.sql]: this looks generic enough to be moved in ut-core

File: api/sql/schema/340$transfer.pagingTT.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+CREATE TYPE [transfer].[pagingTT] AS TABLE( --table type variable used for paging properties
```
Comment (kalinkrustev) [inline api/sql/schema/340$transfer.pagingTT.sql]: this looks generic enough to be moved in ut-core

Comment (venconi91) [inline api/sql/schema/340$transfer.pagingTT.sql]: done

File: api/sql/schema/340$transfer.orderByTT.sql (hunk 1 above)
Comment (venconi91) [inline api/sql/schema/340$transfer.orderByTT.sql]: done

**Pull request merged**